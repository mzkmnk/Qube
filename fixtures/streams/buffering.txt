# 不完全行のバッファリング
Hello, World
Next line starts then
continues and completes here
