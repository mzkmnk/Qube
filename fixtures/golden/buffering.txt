Hello, World
Next line starts then
continues and completes here
